com.unity.render-pipelines.high-definition-config/package.json

```diff
@@ -3,7 +3,6 @@
   "description": "Configuration files for the High Definition Render Pipeline.",
   "version": "10.1.0-preview.24",
   "unity": "2020.2",
-  "unityRelease": "0a20",
```

Comment:
why do you removed the min version? it is required

Reply:
Thanks, it is an incorrect merge issue.
I think git crewed up the history for this file. if I checkout master, then this branch, the line does not disappear locally, but browsing the branch remotely shows the line is removed indeed.
I just pushed 2 commits to fix the history again and if you browse the file remotely, the line has been added back:
https://github.com/Unity-Technologies/Graphics/blob/33975055eb7718a496dfcec749d3507ed02eb4ce/com.unity.render-pipelines.high-definition/package.json#L6
However, the branch diff feature on github still shows it wrong?!

Reply:
your link target hdrp, here it is hdrp-config package.

Reply:
I see that explains everything...
I pushed a new fix on that file.